[2024-07-16 02:46:23,166] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-09-15T00:00:00+00:00 [queued]>
[2024-07-16 02:46:23,208] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-09-15T00:00:00+00:00 [queued]>
[2024-07-16 02:46:23,211] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:23,213] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-16 02:46:23,213] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:23,250] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-15T00:00:00+00:00
[2024-07-16 02:46:23,261] {standard_task_runner.py:52} INFO - Started process 3369 to run task
[2024-07-16 02:46:23,285] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2021-09-15T00:00:00+00:00', '--job-id', '920', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpldrd0qak', '--error-file', '/tmp/tmpz1gy69rm']
[2024-07-16 02:46:23,330] {standard_task_runner.py:77} INFO - Job 920: Subtask second_task
[2024-07-16 02:46:23,562] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2021-09-15T00:00:00+00:00 [running]> on host f3060829e7b4
[2024-07-16 02:46:23,740] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-15T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-15T00:00:00+00:00
[2024-07-16 02:46:23,745] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-16 02:46:23,752] {bash.py:158} INFO - Running command: echo hello world, this is the second task, it will run after first task
[2024-07-16 02:46:23,791] {bash.py:169} INFO - Output:
[2024-07-16 02:46:23,796] {bash.py:173} INFO - hello world, this is the second task, it will run after first task
[2024-07-16 02:46:23,803] {bash.py:177} INFO - Command exited with return code 0
[2024-07-16 02:46:23,974] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20210915T000000, start_date=20240716T024623, end_date=20240716T024623
[2024-07-16 02:46:24,300] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-16 02:46:24,368] {local_task_job.py:146} INFO - Task exited with return code 0
